packages/contracts/package.json

```diff
@@ -16,9 +16,7 @@
     "coverage": "bun test --coverage",
     "cleanup": "bunx --bun rimraf node_modules dist .turbo"
   },
-  "dependencies": {
-    "o1js": "2.2.0"
-  },
+  "dependencies": {},
```

Comment:
I removed this as it's dangerous to set o1js version in two different places. As soon as they get out of sync, two o1js versions will be loaded and it won't work, so better for this folder to reuse whatever o1js is installed elsewhere